docs/email.rst

```diff
@@ -17,25 +17,29 @@ So now you know you aren't emailing real users, but you'd still like to email
 yourself and test email in general. There are a few settings you'll need to
 use.
 
-First, set the ``EMAIL_BACKEND``. This document assumes you're using the SMTP
-mail backend.
+By default kuma is configured to print the email it sends in your console
+when you run it as plain text. That means the full raw source of every email
+will be visible where you run ``foreman start`` or ``./manage.py runserver``
+manually.
 
-::
+In case you want to override that default (not recommended) to have the emails
+really being sent out, you need to override a setting by modifying your
+``.env`` file in the root directory of the kuma source code. Set the
+``EMAIL_URL`` variable in ``.env`` like so::
```

Comment:
I'm a little nervous to even document this ability. Should we leave it un-documented so that only `EMAIL_URL`-savvy (and therefore email-conscious) contributors know how to send real emails?